server/models/Profile.js

```diff
@@ -0,0 +1,59 @@
+const mongoose = require('mongoose');
+
+const dayAvalSchema = new mongoose.Schema({
+    from: String,
+    to: String,
+    isAvailable: {
+        type: Boolean,
+        default: true
+    },
+    price: Number
```

Comment:
Do we have this field so that the sitter can set a different price rate for each of their blocks of availability? Maybe we can name it accordingly to remove any confusion (e.g. `rate` or `totalFee/Price`)?

Reply:
Since we are using this schema for each day of the week, won't it be better to have a `price/rate` as a part of the Profile model so that sitters can set a fixed price for their services?

Reply:
Since we are using this schema for each day of the week, won't it be better to have `price/rate` as a part of the Profile model so that sitters can set a fixed price for their services?